docs/postgres.rst

```diff
@@ -6,7 +6,19 @@ Postgres version 9.4 or later is known to work.
 Set up database
 ===============
 
-The PostgreSQL database used *must* have the correct encoding set, otherwise
+Assuming your PostgreSQL database user is called ``postgres``, create a user
+``synapse_user`` with::
+
+ su - postgres
+ createuser --pwprompt synapse_user
+ su - synapse_user
```

Comment:
this bit (up to line 19) is now unnecessary: the pwprompt allows you to enter a password

Reply:
this is outdated